app/policies/application_policy.rb

```diff
@@ -15,11 +15,23 @@ def index?
   end
 
   def update?
-    @user.admin? ||  @record.user == @user
+    is_admin? ||  @record.user == @user
```

Comment:
totally agree. policies need cleaning up/refactoring.  we have a chore about refactoring MembershipApplicationPolicy, but we really do need to do all of them.  I'll make one.